custom-vu/src/main/kotlin/jces1209/vu/page/CloudIssuePage.kt

```diff
@@ -28,6 +31,31 @@ class CloudIssuePage(
         }
     }
 
+    override fun editDescription(description: String): CloudIssuePage {
+        driver
+            .findElement(By.cssSelector("div[data-test-id = 'issue.views.field.rich-text.description']"))
```

Comment:
I'd avoid requiring a `div`, we don't really mind if it's a `div` or `span` or whatever.
```suggestion
            .findElement(By.cssSelector("[data-test-id = 'issue.views.field.rich-text.description']"))
```